tests/Unit/Application/ClimateLoggerTest.php

```diff
@@ -1,21 +1,22 @@
-<?php
+<?php declare(strict_types=1);
 
 namespace Shopware\Psh\Test\Unit\Application;
 
 use Khill\Duration\Duration;
 use League\CLImate\CLImate;
+use PHPUnit\Framework\TestCase;
 use Shopware\Psh\Application\ClimateLogger;
 use Shopware\Psh\Test\Acceptance\MockWriter;
 
-class ClimateLoggerTest extends \PHPUnit_Framework_TestCase
+class ClimateLoggerTest extends TestCase
 {
     public function test_success_output()
     {
         $cliMateLogger = new ClimateLogger(new CLImate(), new Duration());
         MockWriter::addToClimateLogger($cliMateLogger);
         $cliMateLogger->logSuccess();
 
-        self::assertContains("Executed Successfully", MockWriter::$content);
+        self::assertStringContainsString('Executed Successfully', MockWriter::$content);
```

Comment:
```suggestion
        $this->assertStringContainsString('Executed Successfully', MockWriter::$content);
```

Please fix this in the whole project.

Reply:
Already done, is part of a later pr